### ISSUE TITLE
Just a player

### ISSUE THREAD

**Issue by Khroft** [ironbane/ironbane#4]
Is this able to be opened up and played offline in any capacity?

**Comment by Khroft** [comment]
Maybe not easy, but id jump over some hoops if i needed to.

**Comment by ak65234** [comment]
Yes, the root is the main directory of the project. You run meteor from the command line while your current working directory is the root of the project.
